acceptable-albatrosses/albatrosses_hub/authentication/views.py

```diff
@@ -0,0 +1,103 @@
+from django.contrib.auth import login, logout, authenticate
+from django.contrib.auth.models import User
+from django.shortcuts import render, redirect
+from .forms import RegisterForm, LoginForm
+
+
+# Create your views here.
+def register_page(request):
```

Comment:
It would have been nice to see class-based views here. Basically, the biggest part of this function is boilerplate code that's already provided for you in the classed-based views. You simply give it a template, a form, and *maybe* a custom validation method for those custom error messages you have and you should be good to go. Still, I have a feeling those custom error messages could be handled completely in the form template based on the automated responses when the forms are validated.

It could reduce this entire function to a class definition with three or four defined class attributes.